src/material/tabs/paginated-tab-header.ts

```diff
@@ -246,6 +246,30 @@ export abstract class MatPaginatedTabHeader
     });
   }
 
+  /** A method responsible for sending any change that could affect layout about
+   * items to subscribers.
+   */
+  _getItemChanges() {
+    const itemsTracked = new Set<MatPaginatedTabHeaderItem>();
+    const itemContainerItemChanged = new Subject<void>();
+
+    const observer = new ResizeObserver(() => {
```

Comment:
This needs to be feature detected since it could run in a server environment.

Reply:
Does Angular have their own mechanism for this or can I just do a `window.ResizeObserver` check?

Reply:
You can get it from `this._platform.isBrowser`, although I think I'd prefer to use `typeof ResizeObserver === 'function'` since it would avoid errors on browsers that don't support `ResizeObserver`.